analyses/tumor-purity-exploration/README.md

```diff
@@ -0,0 +1,15 @@
+## Tumor purity exploration module
+
+Author: SJ Spielman `@sjspielman`
+
+This analysis modules explores the tumor purity values and potential correlates.
+
+The full module can be run with `bash run_tumor-purity`.
+
```

Comment:
Can't actually get this to work as a suggestion for code block reasons, but this is my suggestion

The full module can be run with (assuming your current directory is the root of this module):

```sh
bash run_tumor-purity.sh
```

Code block makes this easier to scan. At the very least, need to add `.sh`.